.github/workflows/test.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
         # gain meaning on how job steps use them.
         jupyterlab_version: [2, 3]
         python: [3.6, 3.7, 3.8, 3.9]
-        jupyter_app: [notebook]
+        jupyter_app: [notebook, lab]
```

Comment:
Aren't `server` and `lab` effectively the same? If so we can get of the `include:` below.

Reply:
@manics, under the assumption that you're correct I have removed the extra test case for `server`.

Reply:
Yep, `server` on its own doesn't really do much, `jupyter lab` is what the docs would say.